test/stock-out.spec.js

```diff
@@ -0,0 +1,147 @@
+const { spawnSync } = require('child_process');
+const path = require('path');
+const fs = require('fs-extra');
+//const { once } = require('events');
+const ExcelJS = require('exceljs');
+
+const { stockOutScenario, stockCountScenario } = require('./mocks/mocks');
+const { 
+  setDirToprojectConfig,
+  revertBackToProjectHome
+} = require('./test-utils');
+
+
+describe('Stock out', () => {
+  const workingDir = process.cwd();
+
+  beforeEach(() => {
+    revertBackToProjectHome(workingDir);
+    setDirToprojectConfig();
+  });
+
+  afterEach(async() => {
+    revertBackToProjectHome(workingDir);
+  });
+
+  it('Add stock out summaries test', async() => {
+    const processDir = process.cwd();
+    const workbook = new ExcelJS.Workbook();
+    const childProcess = spawnSync('../../main.js',  stockOutScenario.initScenario);
+
+    if(childProcess.status === 0) {
+      // Check that stock monitoring is initialized and stock count and stock out xform and properties files are generated
+      //const stockMonitoringConfig = path.join(processDir, 'stock-monitoring.config.json');
+
+      const formFiles = fs.readdirSync(path.join(processDir, 'forms', 'app'));
```

Comment:
We know all the new files that should be created. I will suggest to list them
```suggestion
      const createdAppFormFiles = ['stock_count.properties.json', 'stock_count.xlsx'];
      for (const createdAppFormFile of createdAppFormFiles) {
        expect(fs.existsSync(path.join(processDir, 'forms', 'app', createdAppFormFile)), `Expected stock count file to exist at ${createdAppFormFile}, but it was not found.`).toBeTruthy();
      }
```

You could install [jest-expect-message](https://www.npmjs.com/package/jest-expect-message) for custom message. it's up to you...